examples/basic/index.html

```diff
@@ -153,15 +151,23 @@
 }
 
 function onResize() {
```

Comment:
I'd admit it's arguably beyond the scope of the issue you're addressing, but I'm curious:

In your testing, have you seen any prickly issues with the entering/exiting VR, notifications, rotating between landscape to portrait orientations?

<hr>

I've found issues with `resize` and `orientationchange` recently with the following:

1. `resize` _and_ `orientationchange` events not firing reliably on iOS, especially with `<webview>`s (i.e., from within Twitter and Facebook apps)
2. `resize` events not firing properly from
    - entering/exiting VR on my Google Pixel Android phone
    - notifications/doorhangers (e.g., "Add to Homescreen" doorhangers) and other Chrome web notifications (e.g., such as for installing the "Google VR Services" app)

I'm going to try to do some more testing and file some additional bugs (both here in the Polyfill and in the browsers) as needed (if you're _really_ curious, here are some tests I've done across devices with and without the Polyfill): https://docs.google.com/spreadsheets/d/1oZdSMQ4SmmUJdfYccE2MP2fgKMe95fYhVKYS78326zk/edit